packages/ketcher-core/src/application/render/renderers/PeptideRenderer.ts

```diff
@@ -4,8 +4,6 @@ import { BaseMonomerRenderer } from 'application/render/renderers/BaseMonomerRen
 import { MONOMER_SYMBOLS_IDS } from 'application/render/renderers/constants';
 import { KetMonomerClass } from 'application/formatters';
 
-const PEPTIDE_SELECTED_ELEMENT_ID =
-  MONOMER_SYMBOLS_IDS[KetMonomerClass.AminoAcid].selected;
```

Comment:
Not essential, but perhaps it's worth cleaning this `MONOMER_SYMBOLS_IDS` map from `selected` entries now and also removing `selection` svg paths from corresponding components where needed (sometime they're equal to `hover` and sometimes they're not – for whatever reason)
<img width="527" alt="image" src="https://github.com/user-attachments/assets/58dd4b94-25f9-494b-a361-64c34cb1d67a" />
<img width="301" alt="image" src="https://github.com/user-attachments/assets/df345809-15ba-47f3-a8eb-54c1542af190" />